go-controller/pkg/config/config.go

```diff
@@ -314,8 +314,8 @@ type OvnAuthConfig struct {
 	CACert         string `gcfg:"client-cacert"`
 	CertCommonName string `gcfg:"cert-common-name"`
 	Scheme         OvnDBScheme
-
-	northbound bool
+	ElectionTimer  string `gcfg:"election-timer"`
```

Comment:
@fedepaol shouldn't this be a uint seconds value, instead of a string?

Reply:
Good point, will change it

Reply:
Done